src/schematools/permissions/auth.py

```diff
@@ -288,7 +274,7 @@ def get_active_profile_datasets(self, dataset_id: str) -> List[ProfileDatasetSch
         This already checks whether the mandatory user scopes are set.
         """
         if self._all_profiles is None:
-            raise RuntimeError("load_profiles() is not called yet")
+            self._all_profiles = None
```

Comment:
It might be bad idea to remove `RuntimeError` here, old behavior was intentionally breaking when profiles are not loaded yet, avoiding issues with authorization being checked before profiles are ready to use.

New code (not only it is redundant) hides this issue, allowing Profiles to be `None` (not loaded, because Middleware would provide empty list if profiles are not in system).
Or you're planning to change this as well?

Reply:
Changed this to `self._all_profiles = []`. This is needed to make the tests in dso-api work. Discussed this with @vdboor on Slack.